library/entropy_poll.c

```diff
@@ -61,8 +61,8 @@ int mbedtls_platform_entropy_poll(void *data, unsigned char *output, size_t len,
 
     /*
      * BCryptGenRandom takes ULONG for size, which is smaller than size_t on
-     * 64-bit Windows platforms. Ensure len's value can be safely converted into
-     * a ULONG.
+     * 64-bit Windows platforms.Extract entropy in chunks of
+     * len(dependant on ULONG_MAX) size.
```

Comment:
Sorry:
1) Missing space after `platforms.`
2) missing space after `len` before `(dependant on ULONG_MAX)`
3) `dependant` is a noun (e.g. a child), the adjective is `dependent`

And at the least `len (dependent` should be able to fit on the same line as "in chunks of"